net/dynapoint/Makefile

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (C) 2016 LEDE project (Tobias Ilte)
+#
+# This is free software, licensed under the GNU General Public License v3.
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=dynapoint
+PKG_VERSION:=1.0
+PKG_RELEASE:=1
```

Comment:
Please check other packages which do not download external sources but are "self-contained". I think a single version number is sufficient it that case.